CONTRIBUTING.md

```diff
@@ -50,9 +32,26 @@ If you have a feature that you think would be a great addition to the project:
 
 To suggest a feature use the [feature request template](.github/ISSUE_TEMPLATE/feature_request.md).
 
+## Submitting a pull request
+
+1. Check the [issue queue](https://github.com/patternfly/patternfly-elements/issues).  If your pull request does not map to an issue in the queue, please open a new issue and wait for the maintainers to reach out about prioritization.
```

Comment:
@castastrophe do we want them to wait before working on it? I mean, work that is done on Red Hat's time has to go through our prioritization queue but open source code can be worked on whenever. There's no promise of when Red Hatters will have time to review it but we can try to get to it in a reasonable amount of time to encourage contributors?

Reply:
That's fair.  I'd hate for someone to do work on something though that is a "will not fix" kind of thing.